packages/zone/src/virtual.js

```diff
@@ -45,43 +44,10 @@ const defineVirtualExo = (
   return makeInstance();
 };
 
-/**
- * This is a store that is used to check if a value can be stored in a virtual
- * store.  It is not intended to be used for any other purpose.
- */
-const checkVirtualStore = makeScalarBigMapStore('checkVirtualStore');
-
-/**
- * Check if a value can be stored in a virtual store.
- *
- * FIXME: It would be nice if `@agoric/vat-data` exposed a way to do this
- * without having to catch exceptions.
- *
- * @param {unknown} specimen
- * @returns {boolean}
- */
-const isStorable = specimen => {
-  const key = isStorable.name;
-  try {
-    if (checkVirtualStore.has(key)) {
-      // Be defensive in case the store wasn't cleaned up properly.
-      checkVirtualStore.set(key, specimen);
-    } else {
-      // Initialize the store with our specimen.
-      checkVirtualStore.init(key, specimen);
-    }
-    checkVirtualStore.delete(key);
-    return true;
-  } catch (_e) {
-    return false;
-  }
-};
-harden(isStorable);
-
 /** @type {import('.').Stores} */
-export const detachedVirtualStores = Far('virtualStores', {
+export const detachedVirtualStores = harden({
   detached: () => detachedVirtualStores,
-  isStorable,
+  isStorable: isPassable,
```

Comment:
I'm suggesting this change because I believe that anything `Passable` can be stored as a value in a virtual store, so an `isPassable` check will always be equivalent to the virtual `isStorable` in the #7891 code in the left panel above.

@chip @warner Is this true?

Reply:
@mhofman says:
> I think promises are currently an exception... or at least bad things happen if we use promises in virtual stores

Reply:
My hypothesis is that the same thing happens as for `Far` objects --- they get "exported" and an "import" of this internal export gets stored into virtual storage.

Someone should try ;)

Attn @FUDCo @warner @mhofman

Reply:
> Someone should try ;)

I'll gladly write a test for this in #7891 after the current PR lands.